MultiVI_tutorial.ipynb

```diff
@@ -0,0 +1,741 @@
+{
```

Comment:
<p>might be nice to show imputed values and visualize pbmc marker genes</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scvi-tutorials/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>